Runtime/JSArray.cs

```diff
@@ -0,0 +1,41 @@
+using System.Collections;
+using System.Collections.Generic;
+
+namespace NodeApi;
+
+public struct JSArray : IEnumerable<JSValue>, IEnumerable
```

Comment:
Why not `IList<JSValue>` or `ICollection<JSValue>`?

Reply:
Good point! I can add it in a follow up PR.

Reply:
I have added the `IList<JSValue>` - it is incomplete for now. We should see if we need all methods.